qemu/tests/cfg/hotplug_mem_share_discard_data.cfg

```diff
@@ -0,0 +1,35 @@
+- hotplug_mem_share_discard_data:
+    only Linux
+    type = hotplug_mem_share_discard_data
+    virt_test_type = qemu
+    login_timeout = 240
+    start_vm = yes
+    not_preprocess = yes
+    prealloc_mem = yes
+    slots_mem = 4
+    maxmem_mem = 32G
+    mem_fixed = 4096
+    use_mem_mem0 = no
+    mem_devs = 'mem0'
+    guest_numa_nodes = 'node0'
+    size_mem_mem0 = ${mem_fixed}M
+    numa_memdev_node0 = mem-mem0
+    backend_mem = memory-backend-file
+    mem-path = /dev/hugepages
+    target_mems = "plug1"
+    share_mem_plug1 = yes
+    size_mem_plug1 = 1G
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches && echo 1 > /proc/sys/vm/compact_memory"
+    mem-path_plug1 = /dev/hugepages/test_file
```

Comment:
It's being used at the beginning of the test (L33) for deleting the file, in case the latest execution didn't finish, the file probably would not have been deleted at L109, leading the case to fail